lib/nuget-parser/nuspec-parser.ts

```diff
@@ -8,80 +8,126 @@ const debug = debugModule('snyk');
 
 const targetFrameworkRegex = /([.a-zA-Z]+)([.0-9]+)/;
 
-export async function parseNuspec(dep, targetFramework) {
-  return Promise.resolve()
-    .then(() => {
-      const nupkgPath = path.resolve(
-        dep.path,
-        dep.name + '.' + dep.version + '.nupkg',
+async function loadNuspecFromAsync(dep) {
```

Comment:
Good job on changing this function to be async! :)

Can you please add the type that it returns?